docs/platforms/flutter/features/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Features
+description: "Learn more about Sentry's Flutter SDK features."
+sidebar_order: 1
+---
+
+<Note>
+
+Sentry's Flutter SDK enables automatic reporting of errors, messages, and exceptions. The SDK is available on GitHub [`sentry-dart`](https://github.com/getsentry/sentry-dart/).
+
+</Note>
+
+**Features:**
+
+- Automatic native crash error tracking (using both [Android](/platforms/android/) and [iOS](/platforms/apple/)), including
+    - Java, Kotlin, C, and C++ code for Android
+    - ObjC, Swift, and C for iOS
+- Offline storage of events
+- Events [enriched](/platforms/flutter/enriching-events/context/) with device data
+- Breadcrumbs automatically captured:
+    - by the [Flutter SDK](/platforms/flutter/enriching-events/breadcrumbs/#automatic-breadcrumbs)
+    - via the Native SDKs [Automatic Breadcrumbs for Android](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs) and [Automatic Breadcrumbs for iOS](/platforms/apple/usage/#sentryautobreadcrumbtrackingintegration)
+    - as well as `http` with the [Dart SDK](/platforms/dart/)
+- [Release Health](/product/releases/health/) tracks crash free users and sessions
+- Limited support for Flutter Web
+- Under the hood the SDK relies on the [Dart SDK](/platforms/dart/); the minimum required version is `2.8.0` and Flutter SDK version is `1.17.0`.
```

Comment:
@buenaflor I revived this from https://github.com/getsentry/sentry-docs/blob/7577c939491db4a43304c163dbfd0a3b34a87ed0/src/includes/getting-started-primer/flutter.mdx?plain=1 - pls review or update if you have/find a more up to date version

Reply:
I don't think these versions are right, it should be minimum Dart version: `2.17.0` and minimum flutter version: `3.0.0`